src/lib/logger/sanitizer.ts

```diff
@@ -0,0 +1,21 @@
+import type { Sanitizer } from './types'
+
+export const sanitize = (input: string, sanitizer: Sanitizer) =>
+  sanitizer.secrets.reduce<string>((acc, secret) => {
+    if (typeof secret === 'string') {
```

Comment:
Should we explicitly throw exception in case if accidentally we provided a secret not in string format?

Reply:
That's not possible.
This is checked in the application test + typescript types

Reply:
So why we check type here?

Reply:
It's just an extra check. In any case, throwing out errors in the logger is not a common practice.